plugins/raw/README.md

```diff
@@ -1,11 +1,35 @@
-# admin
+# @leancodepl/contractsgenerator-typescript-plugin-raw
 
-This library was generated with [Nx](https://nx.dev).
+Plugin for prepending/appending text to a file generated by Contracts Generator.
 
-## Running unit tests
+## Config
 
-Run `nx test admin` to execute the unit tests via [Jest](https://jestjs.io).
+- `prepend` - text to be added at the beginning of the file
+- `output` - text to be added after content of the plugin specified before this
```

Comment:
Can we also have an example of this? After reading this line a couple of times I still don't 100% understand what this one does or how to exactly use it.